ci-operator/step-registry/baremetal/lab/agent/install/ephemeral/wait-for/baremetal-lab-agent-install-ephemeral-wait-for-commands.sh

```diff
@@ -31,6 +31,17 @@ function get_ready_nodes_count() {
 
 
 function update_image_registry() {
+  # from OCP 4.14, the image-registry is optional, check if ImageRegistry capability is added
+  knownCaps=`oc get clusterversion version -o=jsonpath="{.status.capabilities.knownCapabilities}"`
+  if [[ ${knownCaps} =~ "ImageRegistry" ]]; then
+      echo "knownCapabilities contains ImageRegistry"
+      # check if ImageRegistry capability enabled
+      enabledCaps=`oc get clusterversion version -o=jsonpath="{.status.capabilities.enabledCapabilities}"`
+        if [[ ! ${enabledCaps} =~ "ImageRegistry" ]]; then
+            echo "ImageRegistry capability is not enabled, skip the following tests..."
```

Comment:
```suggestion
            echo "ImageRegistry capability is not enabled, skip the image registry configuration..."
```

Reply:
this is not the step used by the IPI workflow